atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/impl/ExpectationsMetricsReportingTest.java

```diff
@@ -77,7 +84,18 @@ public void mostKvsBytesReadInSingleCallReportedOnReadInfoWithPresentMaximumByte
                 .containsOnly(bytes);
     }
 
+    @Test
+    public void commitLocksRequestsReported() {
+        SnapshotTransaction.reportExpectationsCollectedData(
+                0L,
+                BLANK_READ_INFO,
+                BLANK_COMMIT_LOCK_INFO.withCellCommitLocksRequested(1).withRowCommitLocksRequested(2),
```

Comment:
This is a bit funky, since you've set _both_ parameters on the info.
Either:
1. Test each separately, and use with like the rest of the tests I guess
2. Do one test that sets all properties on the immutable and build the immutable rather than setting with, so you actually get the check that you've set all properties.